tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,154 @@
+<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='0.2.0' profile='20.01'>

```

Comment:
Can you name your version on this model : `version='1.3.0+galaxy0'` ?
So you map your main tool version and add a version for the galaxy wrapper itself